pkgs/development/compilers/vlang/default.nix

```diff
@@ -111,7 +111,7 @@ stdenv.mkDerivation {
     homepage = "https://vlang.io/";
     description = "Simple, fast, safe, compiled language for developing maintainable software";
     license = licenses.mit;
-    maintainers = with maintainers; [ Madouura ];
+    maintainers = with maintainers; [ Madouura, delta231 ];
```

Comment:
please remove `,`

Reply:
Oh, my bad!

Reply:
can you also squash coma removal commit with vlang: 2023.42 -> 2023.44 commit.

this is done so that all commits are always evaluatable, which helps during automatic bisecting

Reply:
I by-mistake squashed with the `maintainer-list` commit :sweat_smile: I hope that will work...